pandas/core/panel.py

```diff
@@ -1528,8 +1528,7 @@ def _extract_axis(self, data, axis=0, intersect=False):
                            'minor_axis': 'columns'})
 
 ops.add_special_arithmetic_methods(Panel, **ops.panel_special_funcs)
-ops.add_flex_arithmetic_methods(Panel, ops._flex_method_PANEL,
-                                flex_comp_method=ops._comp_method_PANEL)
+ops.add_flex_arithmetic_methods(Panel, **ops.panel_flex_funcs)
```

Comment:
why don't we name these the *same* for all classes

Reply:
in fact, why do we need this logic at all, cannot 

``ops.add_flex_arithmetic_method(cls)`` just do this?

or even better
``ops.add_arithmetic_methods(cls)``

Reply:
I like this idea.  SparseSeries does some double-calls that I'll need to sort out to make this work, so let's roll this into the next pass which is Sparse-specific.

Reply:
ok this looks pretty good. I think should take this one step further (can be in another PR). where you then *only* need to call something like

```
ops.add_arithmetic_methods(cls)
```
in each class and push the logic of which flex/special are added to core ops